local-cluster/tests/local_cluster.rs

```diff
@@ -1148,6 +1151,127 @@ fn test_no_voting() {
     }
 }
 
+#[test]
```

Comment:
maybe add an test case for this? https://github.com/solana-labs/solana/pull/9902#issuecomment-627875209

Reply:
Done at test_switch_threshold_across_tower_reload:
https://github.com/solana-labs/solana/pull/10718/files#diff-d848cba681535238849e7e33ed32702bR1323